### ISSUE TITLE
Add ability to share Quark

### ISSUE THREAD

**Issue by AdiChat** [OpenGenus/quark#93]
**I'm submitting a:**  
- [ ] bug report
- [x] feature request

**Description**:

Incorporate the ability to share the link to Quark in Chrome Web Store ([this](https://chrome.google.com/webstore/detail/opengenus-offline-search/lfoloadpfjildomeafpdopahkdaoofbn)) to social media platforms. The sharing icons may appear below the tags. 

Initially, you may support 4 platforms namely: Twitter, Google+, Facebook and LinkedIn.

This will be a convenient feature to share Quark with the user's community. :+1:

**Comment by richa208** [comment]
Yes that will be a great feature for Quark and will make it more known also.
I would like to work on this feature.

**Comment by AnkitaKhurana** [comment]
Yes sharing Quark on social media is the best way to increase the user's community. I would love to work on this, and added to the `Assignees` of this feature.

**Comment by richa208** [comment]
Hi @AnkitaKhurana
I have started working on this. Could you please look into some other issue.
